.travis/config.hpp

```diff
@@ -14,41 +14,49 @@
 
 #if !defined(ARMA_USE_LAPACK)
 #define ARMA_USE_LAPACK
-//// Uncomment the above line if you have LAPACK or a high-speed replacement for LAPACK,
-//// such as Intel's MKL, AMD's ACML, or the Accelerate framework.
+//// Uncomment the above line if you have LAPACK or a high-speed replacement for
```

Comment:
Since we removed the travis config - https://github.com/mlpack/models/pull/10 we can remove the file here as well.